### PR TITLE
Basic coding standards check

### PR DESCRIPTION
**Pull request by thewilkybarkid**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -12,7 +12,8 @@ before_script:
 
 script:
     - nc -z localhost 8080
-    - docker-compose -f docker-compose.yaml run ci
+    - docker-compose -f docker-compose.yaml run ci vendor/bin/phpunit
+    - docker-compose -f docker-compose.yaml run ci vendor/bin/phpcs
```
Comment (thewilkybarkid) [inline .travis.yml]: Would be interesting to try https://docs.travis-ci.com/user/build-stages/share-docker-image/.

File: .gitignore [hunk 2]
```diff
@@ -2,6 +2,11 @@
 /phpunit.xml
 ###< phpunit/phpunit ###
 
+###> squizlabs/php_codesniffer ###
```
Comment (thewilkybarkid) [inline .gitignore]: Not actually added by Flex...

Comment (thewilkybarkid) [reply .gitignore]: Opened https://github.com/symfony/recipes-contrib/pull/456, which would have done it.

File: .travis.yml (hunk 1 above)
Comment (stephenwf) [reply .travis.yml]: Would they run in parallel?

Comment (giorgiosironi) [reply .travis.yml]: Jobs in each stage would be. I am not sure if this is all on the same node or not (as the latter would require `docker-compose up` to be moved together with the `nc` smoke test)

File: phpcs.xml.dist [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+<ruleset xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:noNamespaceSchemaLocation="vendor/squizlabs/php_codesniffer/phpcs.xsd">
+
+    <arg name="basepath" value="."/>
+    <arg name="cache" value=".phpcs-cache"/>
+    <arg name="colors"/>
+    <arg name="extensions" value="php"/>
+    <arg value="s"/>
+
+    <rule ref="PSR2"/>
```
[the remark this answers is not in the document]
Comment (giorgiosironi) [reply phpcs.xml.dist]: If it enforces what is automatically testable from PSR-2, :+1:

**Comment by giorgiosironi** [comment]
https://travis-ci.com/libero/browser/builds/82155931?utm_source=github_status&utm_medium=notification is a good output

**Pull request merged**